inc/class-destination-s3.php

```diff
@@ -130,12 +130,12 @@ class="regular-text"
                                         </legend>
                                         <label for="s3base_multipart">
                                             <input name="s3base_multipart" type="checkbox"
-                                                   checked="checked" value="<?= !empty(
+                                                   value="1" <?= !empty(
                                             BackWPup_Option::get(
                                                 $jobid,
                                                 's3base_multipart'
                                             )
-                                            ) ? '1' : '' ?>">
+                                            ) ? 'checked="checked"' : '' ?>>
```

Comment:
This should use the [checked()](https://developer.wordpress.org/reference/functions/checked/) WordPress function instead of using a manual ternary.